torch/quantization/quantize.py

```diff
@@ -220,6 +226,16 @@ def quantize_qat(model, run_fn, run_args, qconfig_dict=None):
     convert(model)
     return model
 
+def quantize_dynamic(model, run_fn=None, run_args=None, qconfig_dict=None):
+    r"""Converts a float model to dynamic quantized model. Do dynamic training and output a quantized model.
+    """
+    model.eval()
+    model = prepare_dynamic(model, qconfig_dict)
```

Comment:
we should make default qconfig_dict or something. Basically if I call `quantize_dynamic(model)` it should just go and swap out all known layers with default observer settings.

Reply:
Agreed, can we set default qconfig_dict to correspond to nn.Linear (after your change https://github.com/pytorch/pytorch/pull/23212)

Reply:
Will submit a PR on top of https://github.com/pytorch/pytorch/pull/23212 for this change.

Reply:
Added the default qconfig_dict in https://github.com/pytorch/pytorch/pull/24232.